README.md

```diff
@@ -1,33 +1,35 @@
-# Data Converters for TEKTELIC Network Server
-This repository contains the data converters that are to be used on TEKTELIC Network Server for TEKTELIC Sensors. These data converters can be used as a reference for other platforms (such as AWS, TTN, etc).
+# Data Converters for TEKTELIC Devices
+This repository contains the data converters that are to be used on TEKTELIC & TTN Network Server for TEKTELIC Sensors. These data converters can be used as a reference for other platforms.
 
 ### Available Sensors 
```

Comment:
As a general comment, sensor names are in uppercase.  E.g., "COMFORT", rather than "Comfort"